packages/calling-stateful-client/src/CallContext.ts

```diff
@@ -389,6 +391,16 @@ export class CallContext {
     });
   }
 
+  /* @conditional-compile-remove(optimal-video-count) */
+  public setOptimalVideoCount(callId: string, optimalVideoCount: number): void {
+    this.modifyState((draft: CallClientState) => {
+      const call = draft.calls[this._callIdHistory.latestCallId(callId)];
+      if (call) {
+        call.optimalVideoCountFeature.optimalVideoCount = optimalVideoCount;
```

Comment:
I think we should rename this state to be something like `optimalVideoCountState` just so that we don't get confused about invoking the Feature API from the SDK here. We just had this discussion in reviews with the `transfer` work so we will have precedent here.